src/js/angular/ttyg/templates/chat-view.html

```diff
@@ -0,0 +1,28 @@
+<link href="css/ttyg/chat-view.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div class="chat-view">
+    <div class="messages" ng-class="{'hidden': !selectedChat}">
+        <div id="messages-scrollable">
+            <div class="messages-hint text-muted" ng-if="!selectedChat.messages.length">
+                {{ 'ttyg.hint' | translate }}
+            </div>
+            <div ng-repeat="question in selectedChat.messages" class="m-1"
+                 ng-class="{'mt-3': question.role === 'user'}">
+                <span ng-if="question.role === 'user'">User:  </span>
+                <span class="messages-item-content">{{ question.message }}</span>
+            </div>
+            <div onto-loader-new ng-show="loader" size="40"></div>
+        </div>
+    </div>
+    <div class="question">
+        <form>
```

Comment:
This form is meaningless here because it's not used in any way but only as an additional container which is just redundant

Reply:
done